src/test/java/uk/gov/hmcts/reform/preapi/services/RecordingServiceTest.java

```diff
@@ -590,4 +600,88 @@ void undeleteCaptureSessionNotFound() {
         verify(captureSessionService, times(1)).undelete(captureSession.getId());
         verify(recordingRepository, never()).save(recording);
     }
+
+    @DisplayName("New email service should be used on recording created when enabled")
+    @Test
+    void newEmailServiceUsedOnRecordingCreated() {
+        var recordingModel = new CreateRecordingDTO();
+        recordingModel.setId(UUID.randomUUID());
+        recordingModel.setCaptureSessionId(UUID.randomUUID());
+        recordingModel.setVersion(1);
+        var captureSession = new CaptureSession();
+        captureSession.setId(recordingModel.getCaptureSessionId());
+        var booking = new Booking();
+        booking.setCaseId(new Case());
+        captureSession.setBooking(booking);
+        var share = createShare();
+        share.setId(UUID.randomUUID());
+
+        when(
+            recordingRepository.existsByIdAndDeletedAtIsNull(recordingModel.getId())
+        ).thenReturn(false);
+        when(
+            captureSessionRepository.findByIdAndDeletedAtIsNull(
+                recordingModel.getCaptureSessionId()
+            )
+        ).thenReturn(Optional.of(captureSession));
+        when(recordingRepository.findById(any())).thenReturn(Optional.empty());
+        when(recordingRepository.save(any())).thenReturn(new Recording());
+        when(emailServiceBroker.isEnabled()).thenReturn(true);
+        when(emailServiceBroker.getEnabledEmailService()).thenReturn(govNotify);
```

Comment:
This could be moved to the `@BeforeEach` at the top of the class so you don't have to repeat yourself in each test